ansible/roles/de-update-integration-data/tasks/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: update docker-py
+  pip: name="docker-py" state="latest"
+
+- name: pull the latest integrationator image
+  docker_image: name="{{ integrationator_repo }}" tag="{{ docker_tag }}" force=True
+
+- name: update integration data records
+  shell: >-
+    docker run --rm {{ integrationator_repo }}:{{ docker_tag }} -l {{ ldap_host }} -h {{ db_host }}
+    -p {{ db_port }} -d {{ db_name }} -U {{ db_user }} -P {{ db_password }}
```

Comment:
I tried to use the `docker_container` module to start the container, but I couldn't find a way to tell Ansible to wait for the container to exit. Using the shell command fixes that problem.